extras/userdoc/md/documentation/part-4-topologically-structured-networks.md

```diff
@@ -335,9 +351,4 @@ listed in NTUM Section 4.1, are:
 ## References
 
 \[1\]<a id="1"></a>  Hans Ekkehard Plesser and Håkon Enger NEST Topology User
- Manual
-
-## Acknowledgments
```

Comment:
I don't know who Sarah Jarvis is, but I don't think keeping this acknowledgement costs much, does it?
Maybe you can add a "revised by" and the date at the end...